### DISCUSSION TITLE
Keyboard navigation like vim/less

### DISCUSSION THREAD

**Discussion by eggbean** [sumatrapdfreader/sumatrapdf#3748] [Ideas]
I like that j/k can be used for up and down, but it's a bit too slow on the PDFs and ePUB files that I have tried it on. Is there any way to speed it up? It's also a bit jerky (not smooth).

If u and d could be mapped to the same as PgUp and PgDn, it would be nice.

**Comment by GitHubRulesOK** [comment]
Its all in the manual read https://www.sumatrapdfreader.org/docs/Customizing-keyboard-shortcuts

**Reply by eggbean** [reply]
Great, thanks.
